src/EFCore.PG/Scaffolding/Internal/DbDataReaderExtension.cs

```diff
@@ -1,27 +1,35 @@
 ﻿using System.Data.Common;
 using System.Diagnostics;
-using JetBrains.Annotations;
```

Comment:
Am happy that you started to remove them and use built-in. Good work!

Reply:
This will take a *long* time and a lot of work :) But I intend to work on this a lot for EF Core.

Note that for now, in EF Core (and so also EFCore.PG) we're only removing Jetbrains annotations for private/internal APIs. For user-facing public ones we're retaining them for users who don't have the feature etc.